ddl/delete_range.go

```diff
@@ -274,15 +276,16 @@ func insertJobIntoDeleteRangeTable(ctx context.Context, sctx sessionctx.Context,
 			for _, pid := range physicalTableIDs {
 				startKey = tablecodec.EncodeTablePrefix(pid)
 				endKey := tablecodec.EncodeTablePrefix(pid + 1)
-				if err := doInsert(ctx, s, job.ID, pid, startKey, endKey, now); err != nil {
+				if err := doInsert(ctx, s, job.ID, elementID, startKey, endKey, now, fmt.Sprintf("partition table ID is %d", pid)); err != nil {
```

Comment:
```suggestion
				if err := doInsert(ctx, s, job.ID, elementID, startKey, endKey, now, fmt.Sprintf("partition ID: %d", pid)); err != nil {
```

I don't think 'partition table' is an accurate term, it might confuse the customer...using 'partition ID' is better.